src/view/login.js

```diff
@@ -0,0 +1,51 @@
+import Layout from "component/layout";
+
+var getUser = function(acct_id, users) {
+    var acctUser = users.filter(function(u) { return u.accountId() == acct_id; });
```

Comment:
```var account_user``` instead of ```var acctUser```? I'm a big fan of typing a few extra characters over using abbreviations, but I'm sure everyone has their own ideas on that one.